plugins/module_utils/prism/vms.py

```diff
@@ -355,14 +355,17 @@ def _build_spec_boot_config(self, payload, param):
         return payload, None
 
     def _build_spec_gc(self, payload, param):
-        fpath = param["script_path"]
+        if param["script_path"]:
+            fpath = param["script_path"]
 
-        if not os.path.exists(fpath):
-            error = "File not found: {0}".format(fpath)
-            return None, error
+            if not os.path.exists(fpath):
+                error = "File not found: {0}".format(fpath)
+                return None, error
 
-        with open(fpath, "rb") as f:
-            content = base64.b64encode(f.read())
+            with open(fpath, "rb") as f:
+                content = base64.b64encode(f.read())
+        else:
+            content = base64.b64encode(param["script"])
```

Comment:
param of `b64encode ` function should be a bytes-like object

`content = base64.b64encode(param["script"].encode('ascii'))`